### PR TITLE
add node affinity to linkerd-viz

### PR DESCRIPTION
**Pull request by jrsalgado**
Add to Helm linkerd-jaeguer and tracing deployment resources nodeAffinity labels

NodeAffinity was not enabled for Jaeguer Deployments resources

Add to helm values and each of Viz Deployment the labels to add the affinity labels from the shared partials and
allow to be more granular on the manifest setup

cd viz/charts/linkerd-viz/
helm template .

Fixes https://github.com/linkerd/linkerd2/issues/10680

Signed-off-by: Jayro Salgado [jayrosalgado@gmail.com](mailto:jayrosalgado@gmail.com)

### CODE REVIEW

File: viz/charts/linkerd-viz/templates/web.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ spec:
   - name: admin-http
     port: 9994
     targetPort: 9994
+
```
Comment (alpeb) [inline viz/charts/linkerd-viz/templates/web.yaml:31]: Why the empty line?

File: viz/charts/linkerd-viz/values.yaml [hunk 2]
```diff
@@ -35,6 +35,17 @@ linkerdNamespace: linkerd
...
+# [K8S documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) for more information
+nodeAffinity: &default_node_affinity_selector
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: zone
+          operator: In
+          values:
+          - east
+          - west
```
Comment (alpeb) [inline viz/charts/linkerd-viz/values.yaml:48]: Please remove this sample data

**Review by alpeb — COMMENTED** [review]
Thanks @jrsalgado 
Please remember to update the test golden files by running `go test ./... -update`.

**Review by alpeb — COMMENTED** [review]
You need to call `./bin/helm-docs` again

**Comment by reyes256** [comment]
@alpeb something went wrong with the last commit, im working on fixing it.

**Pull request closed without merging**